Assets/Mirror/Runtime/Transport/TelepathyTransport.cs

```diff
@@ -120,5 +117,18 @@ public int GetMaxPacketSize(int channelId)
             // Telepathy's limit is Array.Length, which is int
             return int.MaxValue;
         }
+
+        public override string ToString()
+        {
+            if (server.Active)
+            {
+                return "Telepathy Server port: " + port;
```

Comment:
this would be more robust and future proof:
```cs
return "Telepathy server: " + server.listener.LocalEndpoint;
```
Why you ask?   well:
1) port is a property,  theoretically someone could change the port after the server start, so we would be displaying the incorrect port.
2) LocalEndpoint shows the local ip.  This can be really helpful if we bind to a specific ip address instead of all, and you are trying to troubleshoot connections,  or trying to set up port forwarding.